app/templates/components/event-card.hbs

```diff
@@ -55,9 +55,18 @@
               {{t-var this.tags.[2]}}
             </a>
           {{else}}
-            {{#each this.tags as |tag|}}
-              <a href="#">{{t-var tag}}</a>
-            {{/each}}
+            <a href="{{href-to 'explore' (query-params category=this.event.topic.name event_type=null sub_category=null)}}"
```

Comment:
other way could be add all the queryparams and declared null if they are not needed